test/floki/selector/parser_test.exs

```diff
@@ -22,6 +22,14 @@ defmodule Floki.Selector.ParserTest do
     end
   end
 
+  test "escaped colons in class names" do
+    tokens = tokenize("a.xs\\:red-500")
+
+    assert Parser.parse(tokens) == [
+             %Selector{type: "a", classes: ["xs\\:red-500"], pseudo_classes: []}
```

Comment:
@philss I was thinking here, do you think that this is the expected behavior? Or we actually want to return "xs:red-500", I did this based on the original issue, but maybe the author made a typo in the example code

Reply:
@viniciusmuller I did a quick fix in #464 

I believe the author was trying to find elements with Tailwind classes, which can have colons in it.